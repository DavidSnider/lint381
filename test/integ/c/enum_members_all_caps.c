enum {
    bar,
    BAZ
}

enum Foo_e {
    BAR = 3,
    baz,
    QUX,
}
