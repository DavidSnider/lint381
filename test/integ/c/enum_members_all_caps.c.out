enum_members_all_caps.c:2:5: error: Enum member 'bar' should be all-caps
    bar,
    ^^^
enum_members_all_caps.c:8:5: error: Enum member 'baz' should be all-caps
    baz,
    ^^^
